openapi-cli/src/main/java/io/ballerina/openapi/cmd/BallerinaCodeGenerator.java

```diff
@@ -340,16 +312,22 @@ private List<GenSrcFile> generateClientFiles(Path openAPI, Filter filter, boolea
         }
         List<GenSrcFile> sourceFiles = new ArrayList<>();
         // Normalize OpenAPI definition
-        OpenAPI openAPIDef = normalizeOpenAPI(openAPI, !isResource);
+        OpenAPI openAPIDef = GeneratorUtils.normalizeOpenAPI(openAPI, !isResource);
         // Generate ballerina service and resources.
-        BallerinaClientGenerator ballerinaClientGenerator = new BallerinaClientGenerator(openAPIDef, filter, nullable
-                , isResource);
+        ClientMetaData.ClientMetaDataBuilder clientMetaDataBuilder = new ClientMetaData.ClientMetaDataBuilder();
```

Comment:
Shall we consider renaming the builder inner class as follows?
```suggestion
        ClientMetaData.Builder clientMetaDataBuilder = new ClientMetaData.Builder();
```